web/pages/meetings/index.md

```diff
@@ -5,6 +5,16 @@ author: Erik T. Everson
 
 ----
 
+#### [PlasmaPy "Office" Hours](./office_hours): October 7 & 8 from 2–3 pm EDT
+
+#### *Growing an open source software ecosystem for plasma science*: Mini-conference on November 11 & 12 at [APS DPP meeting](https://engage.aps.org/dpp/meetings/annual-meeting)
+
+----
+
+### Weekly Meetings
```

Comment:
To me it makes more sense to have "Weekly Meetings" as the top most section.  If we're good about keeping the "Upcoming Meetings" up-to-date, then the weekly meeting section could get buried.